src/libs/E2E/API.mock.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable rulesdir/no-api-in-views */
+import _ from 'underscore';
+import Onyx from 'react-native-onyx';
+import Log from '../Log';
+
+/**
+ * A dictionary which has the name of a API command as key, and a function which
+ * receives the api command parameters as value and is expected to return a response
+ * object.
+ */
+const mocks = {
+    BeginSignIn: ({email}) => {
+        const response = require('../E2E/apiMocks/beginSignin.json');
+        response.onyxData.forEach((d) => {
```

Comment:
```suggestion
        response.onyxData.forEach((data) => {
```

We prefer whole words instead of just single characters when possible